public/src/components/account.jsx

```diff
@@ -75,6 +75,6 @@ function mapStateToProps(state) {
   return { auth: state.auth, account_info: state.account_info };
 }
 
-export default withRouter(
+export const ConnectedAccountPage = withRouter(
```

Comment:
is there a reason this was done this way? should I also be doing it for the App container?